tools/python/onnx_test_data_utils.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import glob
+import os
+import sys
+
+import numpy as np
+
+from onnx import numpy_helper
+from onnx import mapping
+from onnx import TensorProto
+
+
+def dump_tensorproto_pb_file(filename):
+    """Dump the data from a pb file containing a TensorProto."""
+
+    tensor = TensorProto()
+    with open(filename, 'rb') as f:
+        tensor.ParseFromString(f.read())
+
```

Comment:
could be tensor = onnx.load_tensor(filename)

Reply:
Nice. Wasn't aware of that helper.